core_backend/tests/api/test_question_dashboard.py

```diff
@@ -1,17 +1,46 @@
+import random
+from functools import partial
+
 import pytest
 from fastapi.testclient import TestClient
 
-from .conftest import mock_dashboard_stats
+from core_backend.app.question_dashboard.schemas import QuestionDashBoard
+
+
+async def mock_dashboard_stats(
+    user_id: int, asession: None, questions: tuple, upvotes: tuple
+) -> QuestionDashBoard:
+    """
+    Function to monkeypatch question_dashboard.routersaz.get_dashboard_stats with
+    random data
+    """
+    return QuestionDashBoard(
+        six_months_questions=[
+            random.randint(questions[0], questions[1]) for _ in range(6)
```

Comment:
My goal was to check for corner cases. I see that you are doing that by including (0, 0) but this may just make it more complex (took me a while to understand it) and I'm unsure of the benefit. 

Can we set a seed to make this deterministic / replicable?

Reply:
I did it that way because it is just too much static code to do it that way, because you need to enter 6 values twice for every case and then check it one by one. Is there any value to make it replicable?